docs/blogs/posts/2024/2024-01-parakeet-tdt.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Unveiling NVIDIA NeMo's Parakeet-TDT -- Turbocharged ASR with Unraveled Accuracy
+author: [Hainan Xu, Nithin Rao Koluguri, Somshubra Majumdar]
+author_gh_user: [hainan-xv, nithinraok, titu1994]
+readtime: 5
+date: 2024-01-25
+
+# Optional: Categories
+categories:
+  - Announcements
+
+# Optional: OpenGraph metadata
+og_title: NVIDIA NeMo Parakeet-TDT
+# og_image: Image for Rich URL previews (absolute URL)
+# og_image_type: Image type (e.g. image/png). Defaults to image/png.
+# page_path: Relative path to the image from the website root (e.g. /assets/images/). If specified, the image at this path will be used for the link preview. It is unlikely you will need this parameter - you can probably use og_image instead.
+description: State of the Art Speech Recognition for Everyone
+---
+
+# Unveiling NVIDIA NeMo's Parakeet-TDT -- Turbocharged ASR with Unraveled Accuracy
+
+Earlier this month, we announced [Parakeet](https://huggingface.co/collections/nvidia/parakeet-659711f49d1469e51546e021), a cutting-edge collection of state-of-the-art ASR models built by [NVIDIA's NeMo](https://nvidia.github.io/NeMo/) toolkit, developed jointly with [Suno.ai](http://suno.ai/). Today, we're thrilled to announce the latest addition to the Parakeet family -- [Parakeet TDT](https://huggingface.co/nvidia/parakeet-tdt-1.1b). Parakeet TDT achieves unraveled accuracy while running more than 70% faster over our previous best model, making it a great choice for powering speech recognition engines in diverse environments.
+
+
+The "TDT" in Parakeet-TDT is short for "Token-and-Duration Transducer", a novel sequence modeling architecture developed by NVIDIA and is open-sourced through [NVIDIA's NeMo](https://nvidia.github.io/NeMo/) toolkit. Our research on TDT models, presented in a [paper](https://arxiv.org/abs/2304.06795) at the ICML 2023 conference, showcases the superior speed and recognition accuracy of TDT models compared to conventional Transducers of similar sizes. 
+
+To put things in perspective, our Parakeet-TDT model with 1.1 billion parameters outperforms similar-sized Parakeet-RNNT-1.1b in accuracy, as measured as the average performance among 9 benchmarks on the [HuggingFace Leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard). Additionally, it achieves an impressive real-time factor (RTF) of 8.5e-3, 75% faster than Parakeet-RNNT-1.1b's RTF of 15e-3. Remarkably, Parakeet-TDT's RTF is even 41% faster than Parakeet-RNNT-0.6b (RTF 12.3), despite the latter having about half the model size.
```

Comment:
This sentence we need to rewrite a bit to mention TDT achieves similar as we shall release another model with avg WER of 6.55, I am also thinking if we do joint post for both these models? @titu1994